assets/js/components/Dashboard/AlertDetails.tsx

```diff
@@ -26,6 +34,44 @@ const AlertDetails: ComponentType = () => {
   const navigate = useNavigate();
 
   const [showModal, setShowModal] = useState(false);
+  const [sortDirection, setSortDirection] = useState(0 as DirectionID);
+  const [modeLineFilterValue, setModeLineFilterValue] = useState(
+    MODES_AND_LINES[0]
+  );
+  const [screenTypeFilterValue, setScreenTypeFilterValue] = useState(
+    SCREEN_TYPES[0]
+  );
+  const [statusFilterValue, setStatusFilterValue] = useState(STATUSES[0]);
+  const [activeEventKeys, setActiveEventKeys] = useState([]);
```

Comment:
I'm honestly not sure if this would be an improvement, but one option that would let you avoid duplicating all these separate state instantiations would be to create the context again, reusing one or more providers from `useScreenplayContext.tsx`—this new context would take priority over the existing one(s) provided higher up in the component tree.

https://reactjs.org/docs/hooks-reference.html#usecontext

> The current context value is determined by the `value` prop of the nearest `<MyContext.Provider>` above the calling component in the tree.

Then, the component that consumes the context values could continue doing so, not caring about whether they're the ones from the top-level provider, or the lower-level AlertDetails-specific one.